### ISSUE TITLE
Lorentz Law

### ISSUE THREAD

**Issue by latot** [RobinKa/maxwell-simulation#16]
Hi hi, I think this is a very very veryyy interesting project, I was thinking would be great include the Lorentz Law in the list of equations, with it the project would be able to simulate all the electro-magnetic aspects :3

https://www.physicsforums.com/threads/derivation-of-lorentz-force-law-strictly-from-maxwells-equations.251261/

Ty.

**Comment by RobinKa** [comment]
Hey, how do you want to include it? Do you want to be able to drop in a point particle and see it moving using the lorentz law?

**Comment by latot** [comment]
Yeah, and not just a point, or maybe, set a point and can put the amount of charge in the point.

The lorentz law is not enough not show the right movement of a charge, is necessary apply all the maxwell equations, the charge will cause electric and magnetic fields, and when are moving will cause new ones, I think would be great add this, we would be able to see signals in precense of charge and how charges move there.
